the-collab-lab/tcl-38-smart-shopping-list
# Ce dm firebase test view

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This PR adds a view that tests reading and writing from/to the database. We created a Pages folder and a FirebaseTest.js file inside of that. The test view includes a form for submitting an item and a list for displaying items added.

## Related Issue
Closes #1 

## Acceptance Criteria




## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before
About to enter "THE VERY LAST TEST" in the input and click "Add item"
![Screen Shot 2022-01-11 at 1 19 12 PM](https://user-images.githubusercontent.com/83732773/149023384-92f5d042-be4f-4781-81a3-e4f4de6a762f.png)


### After
See end of list for "THE VERY LAST TEST"
![Screen Shot 2022-01-11 at 1 19 39 PM](https://user-images.githubusercontent.com/83732773/149023285-f81d7787-c79a-402b-90f4-8c809527055c.png)


## Testing Steps / QA Criteria

Pull down this branch and fill out the form with a new item. Upon submitting the item, it should appear in the list below, and remain in the list upon refresh.
